<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="love">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加收藏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="found2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="found1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>451</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>661</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
